aries-site/src/pages/templates/selector.mdx

```diff
@@ -0,0 +1,93 @@
+import { Notification } from 'grommet';
+import { ShareRounded } from 'grommet-icons';
+import { Example } from '../../layouts';
+import {
+  SimpleSelector,
+  QuickFilter,
+  SupportSelector,
+  TEMP,
+} from '../../examples';
+
+<Example
+  figma="TBD"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SimpleSelector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/Selector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/SelectorGroup.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-core/src/js/components/core/Selector"
+  width="100%"
+>
+  <SimpleSelector />
+</Example>
+
+## Use cases
+
+### Quick Filtering
+
+Enables users to filter data based on high-level visual options to narrow down datasets within tables and lists.
```

Comment:
It feels like this should capture:
- Provides an efficient way to filter data to a specific view that will be useful and relevant to the workflow.

Maybe not the best wording, but given its high visual prominence, they should be useful and relevant.

It feels like the current guidance is redefining "Selector" in some ways (high prominence visual options) / we can clean up wording since "to narrow down datasets" is redundant with "filter data".

Reply:
okay I re-wording lmk what you think